examples/monitoring_and_alerting/prometheus/prometheus.yml

```diff
@@ -0,0 +1,21 @@
+global:
+  scrape_interval:     15s
+  evaluation_interval: 15s
+
+rule_files:
+  - "/etc/prometheus/alerts/woodpecker_alerts.yml"
+
+scrape_configs:
+  - job_name: ctwoodpecker
+    scrape_interval: 30s
+    static_configs:
+        - targets: ['127.0.0.1:1971' ]
+        labels:
```

Comment:
I believe this line and the next aren't indented to the correct level. I think they should be one tab deeper to the right otherwise Prometheus complains:
```
prometheus_1     | level=error ts=2018-12-18T19:58:46.375494481Z caller=main.go:579 err="Error loading config couldn't load configuration (--config.file=/etc/prometheus/prometheus.yml): parsing YAML file /etc/prometheus/prometheus.yml: yaml: line 12: did not find expected '-' indicator"
```